### PR TITLE
CMO - Send message to SPO and SITE

### PR DESCRIPTION
**Pull request by abhayprasadnovartis**
CMO - Send message to SPO and SITE that Shipment is in Preparation stage.

### CODE REVIEW

File: csc-services/services/ShipmentsService.js [hunk 1]
```diff
@@ -54,6 +63,26 @@ class ShipmentsService extends DSUService {
       shipment.uid
     );
 
+    this.sendMessageToEntity(
+      CommunicationService.identities.CSC.SPONSOR_IDENTITY,
+      messagesEnum.ShipmentInPreparation,
+      {
+        orderSSI: order.uid,
+      },
+      'Shipment In Preparation'
```
Comment (paiusCatalin) [inline csc-services/services/ShipmentsService.js:72]: I see you have added a constant with this String. Please use it.

File: csc-services/services/ShipmentsService.js [hunk 2]
```diff
@@ -54,6 +63,26 @@ class ShipmentsService extends DSUService {
...
+    );
+
+    this.sendMessageToEntity(
+      CommunicationService.identities.CSC.SITE_IDENTITY,
+      messagesEnum.ShipmentInPreparation,
+      {
+        orderSSI: order.uid,
+      },
+      'Shipment In Preparation'
```
Comment (paiusCatalin) [inline csc-services/services/ShipmentsService.js:81]: Same here

**Review by paiusCatalin — CHANGES_REQUESTED** [review]
Please fix the mentioned changes in the comments.

**Review by paiusCatalin — APPROVED** [review]
All good

File: csc-services/services/ShipmentsService.js (hunk 1 above)
Comment (abhayprasadnovartis) [reply csc-services/services/ShipmentsService.js]: Update done

File: csc-services/services/ShipmentsService.js (hunk 2 above)
Comment (abhayprasadnovartis) [reply csc-services/services/ShipmentsService.js]: Update done

**Pull request merged**